### PR TITLE
Fixes #14310 - Display muliple host action buttons only if action per…

### PR DESCRIPTION
**Pull request by tbrisker**
…mitted

### CODE REVIEW

File: app/views/hosts/_list.html.erb [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <% hosts.each do |host| %>
       <tr>
         <td class="ca">
-          <%= check_box_tag "host_ids[]", nil, false, :id => "host_ids_#{host.id}", :disabled => !authorized?, :class => 'host_select_boxes', :onclick => 'hostChecked(this)' %>
```
Comment (tbrisker) [inline app/views/hosts/_list.html.erb:20]: No point disabling the checkboxes, this is an expensive check performed for every host (esp. if we wish to check all the relevant permissions) for very little UX value. If none of the actions are allowed, the drop down will simply not show and this checkbox will have no effect.

**Comment by tbrisker** [comment]
Now with 10% less silly typos.

File: app/views/hosts/index.html.erb [hunk 2]
```diff
@@ -1,4 +1,2 @@
-<% if authorized? %>
-  <% title_actions multiple_actions_select, button_group(link_to_if_authorized _("New Host"), hash_for_new_host_path) %>
-<% end %>
+<% title_actions multiple_actions_select, button_group(link_to_if_authorized _("New Host"), hash_for_new_host_path) %>
```
Comment (domcleal) [inline app/views/hosts/index.html.erb]: Should this now be display_link_if... so it doesn't show as greyed out if not authorised?

Comment (tbrisker) [inline app/views/hosts/index.html.erb]: fixed.

**Comment by domcleal** [comment]
Merged as 29c5556, thanks @tbrisker.

**Pull request closed without merging**